src/main/java/edu/hm/hafner/analysis/parser/MsBuildParser.java

```diff
@@ -32,23 +32,21 @@ public MsBuildParser() {
     @Override
     protected Issue createWarning(final Matcher matcher) {
         String fileName = determineFileName(matcher);
+
```

Comment:
Formatierungen nicht ändern. (Die scheinen auch in IntelliJ falsch konfiguriert zu sein).

Reply:
Alles klar.